src/parser/core/modules/CastTime.ts

```diff
@@ -31,52 +32,128 @@ export default class CastTime extends Analyser {
 	}
 
 	private onApplySwiftcast(): void {
-		this.scIndex = this.set('all', 0)
+		this.scIndex = this.setInstantCastAdjustment()
 	}
 
 	private onRemoveSwiftcast(): void {
 		this.reset(this.scIndex)
 		this.scIndex = null
 	}
 
-	/** @deprecated */
-	public setFflogs(actions: number[] | 'all', castTime: number, start: number = this.parser.currentTimestamp, end: number | null = null): number {
-		return this.set(actions, castTime, this.parser.fflogsToEpoch(start), end ? this.parser.fflogsToEpoch(end) : null)
+	/**
+	 * setTimeAdjustment Module-compatibility function
+	 * @deprecated
+	 */
+	public setTimeAdjustmentFflogs(actions: number[] | 'all', adjustment: number, start: number = this.parser.currentTimestamp, end: number | null = null): number {
```

Comment:
oof, i wrote these sigs originally didn't i. that null 😬.

uuh, this should probably be `start: number = curTs, end?: number`.

---

`Adjustment` seems a bit redundant on these methods but i'm not sure which way it should be taken, given `setTime` is inaccurate.

Reply:
I can't think of a better name for the functions either. English major I was not .-.

Reply:
I think setXAdjustment makes sense as a naming scheme, since the methods are configuring when a given cast time adjustment (window) is occurring.